po/sv_SE.po

```diff
@@ -0,0 +1,247 @@
+msgid "Reload Configuration"
+msgstr "Ladda om konfiguration"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:85
+msgid "Terminal Inspector"
+msgstr "Terminalinspektören"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:102
+#: src/apprt/gtk/Window.zig:933
+msgid "About Ghostty"
+msgstr "Om Ghostty"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:107
+msgid "Quit"
+msgstr "Avsluta"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:6
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:6
+msgid "Authorize Clipboard Access"
+msgstr "Tillåt åtkomst till urklipp"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:7
+msgid ""
+"An application is attempting to read from the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Ett program försöker läsa från urklipp. Det aktuella "
+"innehållet visas nedan."
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:10
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:10
+msgid "Deny"
+msgstr "Neka"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:11
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:11
+msgid "Allow"
+msgstr "Tillåt"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:7
+msgid ""
+"An application is attempting to write to the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Ett program försöker skriva till urklipp. Det aktuella "
+"innehållet visas nedan."
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:6
+msgid "Warning: Potentially Unsafe Paste"
+msgstr "Varning: Potentiellt osäker infogning"
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:7
+msgid ""
+"Pasting this text into the terminal may be dangerous as it looks like some "
+"commands may be executed."
+msgstr ""
+"Att klistra in den här texten i terminalen kan vara farligt eftersom det ser ut som "
+"att kommandon kan utföras."
+
+#: src/apprt/gtk/Window.zig:199
+msgid "View Open Tabs"
+msgstr "Huvudmeny"
+
+#: src/apprt/gtk/Window.zig:219
+msgid "View Open Tabs"
+msgstr "Visa öppna flikar"
+
+#: src/apprt/gtk/Window.zig:265
+msgid ""
+"⚠️ You're running a debug build of Ghostty! Performance will be degraded."
+msgstr ""
+"⚠️ Du kör en felsökningsversion av Ghostty! Prestanda kommer att försämras."
+
+#: src/apprt/gtk/Window.zig:681
+msgid "Reloaded the configuration"
+msgstr "Laddade om konfigurationen"
+
+#: src/apprt/gtk/Window.zig:914
+msgid "Ghostty Developers"
+msgstr "Ghostty-utvecklare"
+
+#: src/apprt/gtk/CloseDialog.zig:47
+msgid "Close"
+msgstr "Stäng"
+
+#: src/apprt/gtk/CloseDialog.zig:87
+msgid "Quit Ghostty?"
+msgstr "Vill du avsluta Ghostty?"
+
+#: src/apprt/gtk/CloseDialog.zig:88
+msgid "Close Window?"
+msgstr "Vill du stänga fönstret?"
+
+#: src/apprt/gtk/CloseDialog.zig:89
+msgid "Close Tab?"
+msgstr "Vill du stänga fliken?"
+
+#: src/apprt/gtk/CloseDialog.zig:90
+msgid "Close Split?"
+msgstr "Vill du stänga uppdelat fönster?"
+
+#: src/apprt/gtk/CloseDialog.zig:96
+msgid "All terminal sessions will be terminated."
+msgstr "Alla terminalsessioner kommer att avslutas."
```

Comment:
```suggestion
msgstr "Alla terminalsessioner kommer avslutas."
```

Definitely a point of contention, but "att" is often an unnecessary word. This article is behind a paywall, but covers it nicely: [Språktidningen](https://spraktidningen.se/tips/att-skriva-att-i-onodan/#:~:text=I%20svenskan%20har%20subjunktioner%20ofta,att%20det%20%C3%A4r%20en%20subjunktion.)

The feedback on this one covers quite a few unnecessary "att"s in this translation file